clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy

```diff
@@ -177,13 +171,34 @@ abstract class OpenstackClientProvider {
    * @return List < ? extends Stack >  stacks
    */
   List<? extends Stack> listStacks() {
-    def stacks
-    try {
-      stacks = client.heat().stacks().list()
-    } catch (Exception e) {
-      throw new OpenstackOperationException(e)
+    handleRequest {
+      client.heat().stacks().list()
+    }
+  }
+
+  /**
+   * Get a stack in a specific region.
+   * @param stackName
+   * @return
+   */
+  Stack getStack(String region, String stackName) {
```

Comment:
This duplicates the `getServerGroup(region, stackName` introduced in the clone server group PR. Please consolidate these methods